specification/recoveryservicessiterecovery/resource-manager/Microsoft.RecoveryServices/stable/2018-01-10/service.json

```diff
@@ -12332,11 +12417,6 @@
           "type": "string",
           "readOnly": true
         },
-        "recoveryServicesProviderId": {
```

Comment:
Removal of a property is considered a breaking change. Please do this in a new version.

Reply:
This is no longer required to be exposed to the customer. This is a read-only property, and removing it should not be a breaking change.